renovate.json

```diff
@@ -0,0 +1,147 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>elastic/renovate-config"
+  ],
+  "baseBranches": [
+    "main",
+    "9.7",
+    "9.6",
+    "9.5",
+    "9.4",
+    "9.3",
+    "9.2",
+    "9.1",
+    "8.19"
+  ],
+  "schedule": [
+    "after 1am on monday"
```

Comment:
Do we have any reason to not run it every day? or multiple times a day?

Reply:
The setup of weekly Monday mornings is the Elastic standard and a recommended practice.